aggregator/flush_mgr.go

```diff
@@ -304,3 +308,15 @@ func newBucket(interval time.Duration, scope tally.Scope) *flushBucket {
 func (b *flushBucket) Add(flusher PeriodicFlusher) {
 	b.flushers = append(b.flushers, flusher)
 }
+
+func (b *flushBucket) Remove(flusher PeriodicFlusher) error {
```

Comment:
nit: seems `flushBucket` is not goroutine safe, perhaps it's worth adding that to it's comment on line 294?

Reply:
Sure sounds good, the bucket is always protected by the flush manager lock, can add a comment.